src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationDeletionEventListener.kt

```diff
@@ -0,0 +1,71 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.listeners.probation
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import feign.FeignException
+import io.awspring.cloud.sqs.annotation.SqsListener
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.NOTIFICATION
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.SQSMessage
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.domainevent.DomainEvent
+import uk.gov.justice.digital.hmpps.personrecord.message.processors.probation.ProbationDeleteProcessor
+import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType
+import uk.gov.justice.digital.hmpps.personrecord.service.EventKeys.EVENT_TYPE
+import uk.gov.justice.digital.hmpps.personrecord.service.EventKeys.MESSAGE_ID
+import uk.gov.justice.digital.hmpps.personrecord.service.EventKeys.SOURCE_SYSTEM
+import uk.gov.justice.digital.hmpps.personrecord.service.TelemetryService
+import uk.gov.justice.digital.hmpps.personrecord.service.type.OFFENDER_GDPR_DELETION
+import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.MESSAGE_PROCESSING_FAILED
+
+const val PROBATION_DELETION_EVENT_QUEUE_CONFIG_KEY = "cprdeliusdeleteeventsqueue"
+
+@Component
+@Profile("!seeding")
```

Comment:
This is an accident waiting to happen! Not sure what we can do that is any better though